xbmc/dbwrappers/sqlitedataset.cpp

```diff
@@ -926,15 +941,15 @@ bool SqliteDataset::query(const std::string& query)
   if (!handle())
     throw DbErrors("No Database Connection");
   const std::string& qry = query;
-  int fs = qry.find("select");
-  int fS = qry.find("SELECT");
-  if (!(fs >= 0 || fS >= 0))
+  const size_t fs = qry.find("select");
+  const size_t fS = qry.find("SELECT");
+  if (fs == std::string::npos && fS == std::string::npos)
```

Comment:
fs and fS are not used later, might as well do the lazy evaluation
```suggestion
  if (qry.find("select") == std::string::npos && qry.find("SELECT") == std::string::npos)
```
though I don't know if low or high cap is the most common in the codebase.

Reply:
Sure, will change that as well. I think upper case is used more.